open-amp/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2018 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if(CONFIG_OPENAMP)
+set(WITH_ZEPHYR 1)
+set(WITH_ZEPHYR_LIB 1)
+if (CONFIG_OPENAMP_MASTER)
+  set(WITH_VIRTIO_DRIVER 1)
+else()
+  set(WITH_VIRTIO_DRIVER OFF CACHE BOOL "" FORCE)
+endif()
+if (CONFIG_OPENAMP_SLAVE)
+  set(WITH_VIRTIO_DEVICE 1)
+else()
+  set(WITH_VIRTIO_DEVICE OFF CACHE BOOL "" FORCE)
+endif()
+if (CONFIG_OPENAMP_WITH_DCACHE)
+  set(WITH_DCACHE_VRINGS 1)
+  set(WITH_DCACHE_BUFFERS 1)
+endif()
+set(WITH_LIBMETAL_FIND OFF CACHE BOOL "" FORCE)
+set(WITH_PROXY OFF CACHE BOOL "" FORCE)
+
+#set(LIBMETAL_INCLUDE_DIR ${ZEPHYR_BASE}/../libmetal/lib/include)
+#set(LIBMETAL_LIB ${ZEPHYR_BASE}/../libmetal/lib)
```

Comment:
So if this works w/o these lines, why are they in the original?

Reply:
Seems that these lines have been introduced by Kumar a long time ago, and now useless.
i propose to remove them here.
i need to figure out the reason and propose a PR in Zehyr